frontend/app/services/zuora/ZuoraRestService.scala

```diff
@@ -32,12 +33,16 @@ class ZuoraRestService(config: ZuoraApiConfig) extends LazyLogging {
     }
   }
 
+  val productCatalogSupplier = new FutureSupplier[ProductCatalog](productCatalog)
```

Comment:
@vivekkr @tudorraul this is where you can start grabbing the Zuora Product Catalog now